### PR TITLE
CVE-2019-14882 update

### PR DESCRIPTION
**Pull request by darunesh-RH**

### CODE REVIEW

File: 2019/14xxx/CVE-2019-14882.json [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        ]
+    },
+    "references": {
+        "reference_data": [
+            {
+                "url": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2019-14882",
+                "name": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2019-14882",
+                "refsource": "CONFIRM"
+            }
+        ]
```
Comment (iamleot) [inline 2019/14xxx/CVE-2019-14882.json]: Please add a reference to upstream security advisory:

 https://moodle.org/mod/forum/discuss.php?d=393585#p1586747

File: 2019/14xxx/CVE-2019-14882.json [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    },
+    "description": {
+        "description_data": [
+            {
+                "lang": "eng",
+                "value": "A vulnerability was found in Moodle 3.7 to 3.7.2, 3.6 to 3.6.6, 3.5 to 3.5.8 and earlier where an open redirect existed in the Lesson edit page."
```
Comment (iamleot) [inline 2019/14xxx/CVE-2019-14882.json]: Please change the description to mention fixed versions:

 3.7.3, 3.6.7 and 3.5.9

**Comment by cve-team** [comment]
Just waiting on implementation of iamleot's comments Dhananjay and we'll get that merged. Thanks!

**Comment by cristina479** [comment]
Red Hat CNA,

Can you address the comments in this PR in order to publish this CVE ID to the CVE List?

Thank you,
Cristina

File: 2019/14xxx/CVE-2019-14882.json (hunk 1 above)
Comment (darunesh-RH) [reply 2019/14xxx/CVE-2019-14882.json]: Updated flaw with the above mentioned reference.

File: 2019/14xxx/CVE-2019-14882.json (hunk 2 above)
Comment (darunesh-RH) [reply 2019/14xxx/CVE-2019-14882.json]: updated description with missing info.

File: 2019/14xxx/CVE-2019-14882.json [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+                "name": "https://moodle.org/mod/forum/discuss.php?d=393585#p1586747",
+                "url": "https://moodle.org/mod/forum/discuss.php?d=393585#p1586747"
+            }
+        ]
+    },
+    "description": {
+        "description_data": [
+            {
+                "lang": "eng",
+                "value": "A vulnerability was found in Moodle 3.7 to 3.7.2, 3.6 to 3.6.6, 3.5 to 3.5.8 and earlier where an open redirect existed in the Lesson edit page."
```
Comment (iamleot) [inline 2019/14xxx/CVE-2019-14882.json]: This is still not updated as suggested: no fixed versions are referred in the description.
Please change:

````
[...] 3.7 to 3.7.2, 3.6 to 3.6.6, 3.5 to 3.5.8 [...]
````

to:

````
[...] 3.7 to 3.7.3, 3.6 to 3.6.7, 3.5 to 3.5.9 [...]
````

File: 2019/14xxx/CVE-2019-14882.json (hunk 2 above)
Comment (iamleot) [reply 2019/14xxx/CVE-2019-14882.json]: That's still not the case.  I have added another revision, please address that!

Thanks!

**Pull request merged**